chart/values.yaml

```diff
@@ -161,6 +161,25 @@ services:
       start: 20000
       end: 20008
 
+# CPU and memory resources can be configured via the `resources` tree when features.cpu_limits.enabled
+# or features.memory_limits.enabled are set respectively. Each setting covers both limit and request
+# settings for their resource type.
+#
+# The helm chart includes default memory limits for all processes, and some explicit requests. When no
+# request size is specified, a default is calculated as a percentage of the limit, but at least some
+# minimum threshold, but never more than the limit itself. See the features.memory_limits setting to
+# finetune this algorithm.
+#
+# All values are integers; cpu values are in millicpus (m) and memory is in megabytes (Mi).
+#
+# More information about the `resources` structure can be found in the config/resources.yaml file
+# inside this helm chart.
+
+resources:
+  diego-cell:
+    garden:
+      garden: {memory: {limit: 2048, request: 2048}}
```

Comment:
This default seems small, I think it should be left to auto by default.
If we keep this non-prod item as a default, there need to be clear instructions on how to remove the limit and request.

Reply:
Possible workaround to get `auto`: Set garden/rep process lists in `config/job.yaml` to empty.
Cannot set to nil, this picks up defaults.

TODO: Testing the workaround.

Reply:
It feels weird to not specify a limit/request for this one pod when you explicitly ask for limits to be enabled.

I intentionally copied these 2 container settings into `values.yaml` so that they are kind-of in the face of the user.

I would rather set `features.memory_limits.enabled=false` by default and put a comment directly above that explains the need to configure the diego-cell size.